### DISCUSSION TITLE
I'm seeing red snapshot age indicators for instances on VMs but not on physical servers

### DISCUSSION THREAD

**Discussion by dennisa72295** [trimble-oss/dba-dash#641] [Q&A] [answered]
![Capture](https://github.com/trimble-oss/dba-dash/assets/85762105/52ee871c-f45a-4426-9c13-2a2f945a5fac)
All of my SQL instances on VMs are showing snapshot age of more than 2 days.  I'm not sure what this is telling me nor what to do about it.  All the snapshots have the same date and time.

**Comment by DavidWiseman** [comment]
Is it just the Drivers collection showing issues?  This collection runs daily at midnight by default.  It's possible the instance was offline at the time of collection or some other issue prevented the data from being collected.  You can adjust the thresholds at which it warns the data is out of date by clicking the links to configure Instance or Root in the grid.  

To troubleshoot why the data hasn't been collected:
* Check the DBA Dash ErrorLog tab to see if there are any errors reported for this collection
* If you don't find anything.  Check the Logs folder where the DBA Dash agent is installed.  (Or click view service log in the service config tool).  In some cases, errors won't make it into the repository DB to report on in the GUI but they should be written to the Logs folder.
* Check the config:
  * Verify the No WMI option isn't checked in the service config tool.  
  * Verify the service account has access to run WMI queries (local admin)  https://dbadash.com/docs/help/wmi/
  * It's possible that the schedule could have been disabled or altered in the config - unlikely you would have done this by accident though.  https://dbadash.com/docs/help/schedule/

Hope this helps,

David

**Comment by dennisa72295** [comment]
David,
Yes it is only the drivers collection.  The No WMI is checked on all my instances (due to security policies not allowing some accounts to have admin rights on server).
I’ll check the logs and see if I find anything.  It seemed odd that they all happened at the same time, but nothing had changed except the age of the snapshot since Monday when it triggered.

Dennis

**Comment by DavidWiseman** [comment, marked as answer]
With No WMI checked the collection won't run - the Drivers collection is only done through WMI.  It must have been enabled previously to allow the collection to run.  For your other instances if the collection had never run then you wouldn't get a warning that it's out of date. If you click the Configure Instance/Root link you can disable the alert for the collection.  

It's a bit disappointing that it needs local admin for it to work.  I've been down a few rabbit holes trying to get it to work but I never had any success.  I think it might work as a non-admin user against the local computer but not against a remote computer.

Most of the data is collected using SQL queries so you don't lose too much by having it disabled.  

https://dbadash.com/docs/help/wmi/

**Comment by dennisa72295** [comment]
David,
Thanks. That was it.  I disabled the collection and it cleared the “alert”.  Everything is green now!
Dennis
